### PR TITLE
Update aggregatable report schema for coordinator origin

### PR DESCRIPTION
**Pull request by linnan-github**

### CODE REVIEW

File: schema/aggregatable_report.schema.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
   "description": "An Attribution Reporting aggregatable report",
   "type": "object",
   "properties": {
-    "aggregation_coordinator_identifier": {
-      "type": "string"
+    "aggregation_coordinator_origin": {
```
Comment (apasel422) [inline schema/aggregatable_report.schema.json:8]: The "required" section below also needs to be updated.

Comment (linnan-github) [reply schema/aggregatable_report.schema.json:8]: Updated, thanks.

**Review by apasel422 — APPROVED** [review]

**Comment by apasel422** [comment]
Wait, is that field required actually?

**Comment by linnan-github** [comment]
> Wait, is that field required actually?

Yes it is, when the feature is enabled.

**Pull request merged**